Documentation/requirements.md

```diff
@@ -0,0 +1,33 @@
+# Requirements
+
+Depending on the [platform](Platforms.md) that you are targeting, MonoGame has different sets of requirements.
+
+## For desktop platforms
+
+MonoGame requires a .Net Core SDK (3.1 or up) installation.
+
+You can either install it [independently](https://dotnet.microsoft.com/download/dotnet-core), or by selecting the .Net Core payload when installing Visual Studio 2019 (version 15.4 and up required).
+
+When it comes to IDE, Visual Studio 2019, Visual Studio Code, and Visual Studio 2019 for Mac are supported (alternatively, you can work directly from the CLI with your code editor of choice).
```

Comment:
Hmm, I'm not sure about this line. We don't really do anything to support VS Code, so it's not more or less supported than, say, JetBrains Rider. Even for VS and VS for Mac we only provide templates.

Or should we mention this so we don't have to troubleshoot issues with other IDE's?

Reply:
Really don't know what's best here. I don't if removing Code will expose us to trivial questions like "is Code supported?" or if Code/CLI users are proficient enough to assume that it is. But yeah, the question is the same for any code editor.

Reply:
Maybe what we should be noting is that VS2017 and prior are not supported instead?